CMakeLists.txt

```diff
@@ -66,7 +66,11 @@ if(_ansakRoot)
 endif()
 
 ##############################################################################################################
-set( _useUnitTests 1 )
+set( _useUnitTests 0 )
+if(TEST_ANSAKSTRING OR _ansakRoot)
```

Comment:
This style matches the best practices mentioned in Professional CMake.  It allows the consuming repo to enable unit tests in chosen submodules, if so desired.  By default they are only enabled in the top-most repo (so when AnsakString is built stand-alone, they are enabled).